java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -382,6 +384,39 @@ public T getdel(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Gets the value associated with the given <code>key</code>. <code>GETEX</code>is similar to
+     * {@see #get}.
+     *
+     * @since Redis 6.0.6.
+     * @see <a href="https://redis.io/docs/latest/commands/getex/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return Command Response - If <code>key</code> exists, return the value of the <code>key</code>
+     *     . Otherwise, return <code>null</code>.
```

Comment:
Will spotless let you put the period one line up? Or move the "\<code>key\</code>" string down so that the period isn't separated? If not no worries

Reply:
yeah spotless hates me unfortunately